server/apps/postgres-app/src/main/java/org/apache/james/PostgresJamesServerMain.java

```diff
@@ -116,7 +125,20 @@ static GuiceJamesServer createServer(PostgresJamesConfiguration configuration) {
         return GuiceJamesServer.forConfiguration(configuration)
             .combineWith(new UsersRepositoryModuleChooser(new PostgresUsersRepositoryModule())
                 .chooseModules(configuration.getUsersRepositoryImplementation()))
-            .combineWith(BlobStoreModulesChooser.chooseModules(configuration.blobStoreConfiguration()))
+            .combineWith(chooseBlobStoreModules(configuration))
             .combineWith(POSTGRES_MODULE_AGGREGATE);
     }
+
+    private static List<Module> chooseBlobStoreModules(PostgresJamesConfiguration configuration) {
+        ImmutableList.Builder<Module> builder = ImmutableList.<Module>builder()
+            .addAll(BlobStoreModulesChooser.chooseModules(configuration.blobStoreConfiguration()))
+            .addAll(BlobStoreCacheModulesChooser.chooseModules(configuration.blobStoreConfiguration()));
```

Comment:
We don't have the blob_cache table yet with postgresql... Is that pertinent here?
Should we add a task for the cache with Postgresql?

Reply:
> We don't have the blob_cache table yet with postgresql... Is that pertinent here?

On that, right now we need CacheDisabledModule which tags the `@Named(MetricableBlobStore.BLOB_STORE_IMPLEMENTATION)` so the blob store binding can work and IT pass. It is how `BlobStoreModulesChooser` and `BlobStoreCacheModulesChooser` designed, I can not against it.

But we enforce disable blob cache configuration for postgres-app for now, so the binding will be fine. Don't worry :-)

>Should we add a task for the cache with Postgresql?

cc @chibenwa do you want it or not?

Reply:
I think it would make sense to have a task for the cache yeah, I think like with cassandra it could be beneficial?

Reply:
>> Should we add a task for the cache with Postgresql?

> cc @chibenwa do you want it or not?

No it do not make sense as headers are a bytea on the message table

Blob store cache thing is a clumsy design around the simple fact that we wanted metadata for distributed app to be stored in Cassandra.

Reply:
Alright... Let's remove this then and just enforce directly the no blob store cache bindings :)